src/coreclr/inc/crsttypes.h

```diff
@@ -131,8 +131,7 @@ enum CrstType
     CrstUnwindInfoTableLock = 113,
     CrstVSDIndirectionCellLock = 114,
     CrstWrapperTemplate = 115,
-    CrstFrozenObjectHeap = 116,
```

Comment:
The removal of CrstFrozenObjectHeap does not looks right.

Looks like you are not synced up to latest main.

Reply:
I did sync and had just assumed someone forgot to run the tool. Let me confirm again.

Reply:
I don't see "FrozenObjectHeap" anywhere in the def file. I've no idea why it was still around. The latest is at https://github.com/dotnet/runtime/blob/main/src/coreclr/inc/CrstTypes.def.

Reply:
Looks like it was added in #49576 without updating the .def file

Reply:
Looks like https://github.com/dotnet/runtime/pull/75573 just renamed things but never updated the def file.

Reply:
Sorry, I didn't realize it's auto-generated now, last time I edited it it was not 😞 Can we add `_generated.` suffix to it?

Reply:
> last time I edited it it was not

This has always been an auto-generated file.

> Can we add _generated. suffix to it?

Sure.